src/js/controller/model.ts

```diff
@@ -71,6 +71,7 @@ export type PlayerModelAttributes = {
     nextUp: PlaylistItem;
     nextUpDisplay: boolean;
     pauseReason: PauseReason;
+    pip: boolean;
```

Comment:
this should already be on the model. Issue with base branch?

Reply:
merged master fixes this issue but my change now appears on master I think I mat have made a mistake here looking into it 👍

Reply:
looks like I didn't have an up to date master and Jong had already fixed this bug with the same change I'll delete the PR 👍